From 1c888d03dbe7d0961324a39c401112ad57ab9fe7 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 24 Jun 2022 11:25:23 +0300
Subject: [PATCH 4/6] fdts: s32cc: Remove the usage of 'device_id' property

This is not a standard property and the same can be achieved by
looking to the alias pointing to SerDes or PCIe node.

Issue: ALB-8357
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 13 +++----------
 1 file changed, 3 insertions(+), 10 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index 544b697f7..4da305dd0 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -38,8 +38,9 @@
 		spi5 = &spi5;
 		spi6 = &qspi;
 		serdes0 = &serdes0;
-		pcie0 = &pcie0;
-		pcie1 = &pcie1;
+		serdes1 = &serdes1;
+		pci0 = &pcie0;
+		pci1 = &pcie1;
 		mmc0 = &usdhc0;
 		clks = &clks;
 		serial0 = &uart0;
@@ -818,9 +819,7 @@
 			reg-names = "dbi", "config";
 			#address-cells = <3>;
 			#size-cells = <2>;
-			device_type = "pci";
 			serdes-handle = <&serdes0>;
-			device_id = <0>;
 			nvmem-cells = <&siul2_0_nvram>;
 
 			ranges =
@@ -846,8 +845,6 @@
 			reg-names = "ss", "dbi";
 			#address-cells = <3>;
 			#size-cells = <2>;
-			device_type = "pci-generic";
-			device_id = <0>;
 			num-lanes = <2>; /* supports max 2 lanes */
 			clocks = <&clks S32GEN1_SCMI_CLK_SERDES_REF>;
 		};
@@ -867,10 +864,8 @@
 			reg-names = "dbi", "config";
 			#address-cells = <3>;
 			#size-cells = <2>;
-			device_type = "pci";
 			serdes-handle = <&serdes1>;
 			nvmem-cells = <&siul2_0_nvram>;
-			device_id = <1>;
 			ranges =
 				/* downstream I/O, 64KB and aligned naturally just
 				 * before the config space to minimize fragmentation
@@ -894,8 +889,6 @@
 			reg-names = "ss", "dbi";
 			#address-cells = <3>;
 			#size-cells = <2>;
-			device_type = "pci-generic";
-			device_id = <1>;
 			num-lanes = <1>; /* supports 1 lane */
 			clocks = <&clks S32GEN1_SCMI_CLK_SERDES_REF>;
 			status = "okay";
-- 
2.17.1

